projects/adin1300_dual/a10soc/system_project.tcl

```diff
@@ -0,0 +1,131 @@
+
+source ../../scripts/adi_env.tcl
+source ../../scripts/adi_project_intel.tcl
+
+adi_project adin1300_dual_a10soc
+
+source $ad_hdl_dir/projects/common/a10soc/a10soc_system_assign.tcl
+
+###############################################################################
+# Triyng to use other pins, this issue migth not be true in the future
```

Comment:
This comment doesn't apply for this board, as pins LA01 and LA05 are not used in the design

Reply:
Done.